GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs

```diff
@@ -830,7 +829,7 @@ public void SetSpellToCast(ISpell s, Vector2 location, IAttackableUnit unit = nu
         /// TODO: Remove Target class.
         public void SetTargetUnit(IAttackableUnit target, bool networked = false)
         {
-            if (target == null || target.IsDead || !target.Status.HasFlag(StatusFlags.Targetable))
+            if (target == null || target.IsDead || !target.Status.HasFlag(StatusFlags.Targetable) || MovementParameters != null)
```

Comment:
Remove this, add a `Target GameObject` to the MovementParameters and use that for the targeted forced movements (dashes).